1.9/administration/installing/custom/remove-a-node.md

```diff
@@ -0,0 +1,66 @@
+---
+post_title: Removing Agent Nodes
+nav_title: Remove Node
+menu_order: 801
+---
+
+You can remove agent nodes from an active DC/OS cluster. 
+
+When you change Mesos attributes (`⁠⁠⁠⁠/var/lib/dcos/mesos-slave-common`⁠⁠⁠⁠) or resources (⁠⁠⁠⁠`/var/lib/dcos/mesos-resources`⁠⁠⁠⁠), you have to remove the agent node and re-register it with the master under a new UUID. The master will then recognize the new attributes and resources specification.
+
```

Comment:
I think we need to mention at the top that you can use one of the two methods: maintenance windows OR killing mesos-agent with USR1 signal. You don't need to do both. And they have reasons why you might want to do them that need to be explained:
- maintenance windows allows draining multiple nodes at the same time from outside the cluster without SSH access.
- draining by kill signal is easier to automate with tools like ansible/chef/puppet that deal that can execute tasks on nodes in parallel similar to how you add nodes with ansible/chef/puppet.